bookkeeping-api.sh

```diff
@@ -0,0 +1,39 @@
+package: bookkeeping-api
+version: "%(tag_basename)s"
+tag: "technical/O2B-677/grpc-poc"
+requires:
+  - grpc
+build_requires:
+  - "GCC-Toolchain:(?!osx)"
+  - CMake
+  - alibuild-recipe-tools
+source: https://github.com/AliceO2Group/Bookkeeping
+incremental_recipe: |
+  cmake --build . -- ${JOBS+-j $JOBS} install
+  mkdir -p $INSTALLROOT/etc/modulefiles && rsync -a --delete etc/modulefiles/ $INSTALLROOT/etc/modulefiles
+---
+
+case $ARCHITECTURE in
+  osx*)
+    # If we preferred system tools, we need to make sure we can pick them up.
+    [[ ! $GRPC_ROOT ]] && GRPC_ROOT=`brew --prefix grpc`
+    [[ ! $OPENSSL_ROOT ]] && OPENSSL_ROOT_DIR=$(brew --prefix openssl@1.1)
+
+    SONAME=dylib
```

Comment:
The `SONAME` variable doesn't seem to be used anywhere, and it's not exported. Can you remove it?

Reply:
I think that I copied it blindly from `o2.sh`. I'll remove it.